resources/assets/js/clipboard.js

```diff
@@ -43,7 +43,7 @@ window.clipboard = () => {
             textArea.style.cssText =
                 "position:absolute;top:0;left:0;z-index:-9999;opacity:0;fontSize:12pt;";
 
-            document.body.append(textArea);
+            this.$el.append(textArea);
```

Comment:
Will this cause issues with the alpine update since `$el` is now `$root`? :thinking:

Reply:
If so, may be worth adding a possible TODO or check, to cater for those projects that may have alpine updated first

Reply:
Yes it possibly will affect that. I'll add a card.

Reply:
Double checked and the alpine upgrade works with both `$el` and `$root` (most modern browsers don't even get to that part anyway because they use the `navigator.clipboard` method instead. I've added a TODO ayway for consistency since it probably should be `$root`